pkg/api/types.go

```diff
@@ -3610,8 +3613,78 @@ type Event struct {
 	// Type of this event (Normal, Warning), new types could be added in the future.
 	// +optional
 	Type string
+
```

Comment:
as I understand the new event will superseed the old API. Can we move the internal types into pkg/apis/events?